content/blog/eng/v-protetto14-4-2023.md

```diff
@@ -0,0 +1,93 @@
+---
+authors: ["engineering"]
+comments: true
+date: "2023-04-18"
+draft: false
+share: true
+categories: [Venerdì Protetto]
+title: "Venerdì Protetto 14/4/2023"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/logo_engineering.png'
+toc: false
+---
+
+# Venerdì Protetto 2023-04-14
+
+This page contains the abstracts of the talks held during the Venerdì Protetto on April 14, 2023. 
+
+## Backstage 
+
+### An introduction to Backstage, managing the chaos
+
+<sup>by [Lee Mills](https://www.linkedin.com/in/codetoy/)</sup>
+
+At the beginning of this year, our engineering teams adopted a new tool: Backstage, an open-source platform for building developer portals created at Spotify. During this Venerdì Protetto, we had the opportunity to host Lee Mills, a member of the Spotify engineering team. Here is a word from our guest:  
+
+> At Spotify, we've always believed in the speed and ingenuity that comes from having autonomous development teams. But as we learned firsthand, the faster you grow, the more fragmented and complex your software ecosystem becomes. And then everything slows down again. Introducing Backstage, a developer portal designed to tame the chaos, bring consistency, and do it all while empowering autonomy and you to make decisions.
+>
+
+During the talk, we discussed the following points:
+
+-   Introduction to Backstage
+
+-   Extension points and potential of Backstage
+
+-   Invitation to developers to contribute to the project
+
+-   Development road map and new intriguing features to expect in future releases
+
+To learn more about Backstage, visit [the official Backstage website](https://backstage.io/docs/overview/what-is-backstage/ "https://backstage.io/docs/overview/what-is-backstage/").
+
+
+$${\color{lightgrey}***}$$ 
```

Comment:
Is this a leftover? It doesn't work. Also, it's present below too.

Reply:
Oops, I removed the dinkus.